src/opcode.rs

```diff
@@ -1660,14 +1660,16 @@ opcode!(
         /// previously registered buffer. The buffer need not be aligned with the start of the
         /// registered buffer.
         buf_index: Option<u16> = None,
+        dest_addr: *const libc::sockaddr = core::ptr::null(),
```

Comment:
would you like to add some docs and tests?

Reply:
Ouch... will try to :)